onnx/version_converter/adapters/group_normalization_20_21.h

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) ONNX Project Contributors
+
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// Adapter for GroupNormalization in default domain from version 20 to 21
+
+#pragma once
+
+#include <memory>
+
+#include "onnx/version_converter/adapters/adapter.h"
+
+namespace ONNX_NAMESPACE {
+namespace version_conversion {
+
+class GroupNormalization_20_21 final : public Adapter {
+ public:
+  explicit GroupNormalization_20_21() : Adapter("GroupNormalization", OpSetID(20), OpSetID(21)) {}
+
+  void transform_input(std::shared_ptr<Graph> graph, Node* node, int64_t input_id) const {
+    // Perform following sequence of ops on input, effect is similar to numpy.repeat()
+
+    //   -- Shape<start=1,end=2>() -- Div(Shape_out, num_groups)
+    //  /                                             |
+    // --  Reshape(,[-1,1]) ------------- Expand(Reshape_out,[1,Div_out]) -- Reshape(Expand_out, [-1])
+
+    Symbol kShape("Shape");
+    Node* shape = graph->create(kShape);
```

Comment:
minor nit: May be naming this `C` instead of `shape` would help clarify what this is intended to be.

Reply:
It's actually not `C` here yet. The adapter will change it to `C` eventually.

Reply:
Scratch the above. I updated it to `C`.